cmake/OpenCL.cmake

```diff
@@ -1,30 +1,19 @@
 option (ENABLE_OPENCL "Use OpenCL optimizations" ON)
 
 set (OPENCL_AMDSDK /opt/AMDAPPSDK-3.0 CACHE PATH "Installation path for the installed AMD OpenCL SDK, if used")
+set (OPENCL_NVIDIASDK "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.0" "Installation path for the installed NVIDIA CUDA Toolkit, if used")
 
 # Options used when building the project
-find_library (LIB_OPENCL OpenCL)
-if (NOT LIB_OPENCL)
-    # Attempt with AMD's OpenCL SDK
-    find_library (LIB_OPENCL OpenCL PATHS ${OPENCL_AMDSDK}/lib/x86_64/)
-    if (NOT LIB_OPENCL)
-        set (ENABLE_OPENCL OFF)
-    else ()
-        # Found, set the required include path.
-        set (OPENCL_INCLUDE_PATH ${OPENCL_AMDSDK}/include CACHE PATH "AMD OpenCL SDK Header include path")
-        set (OPENCL_COMPILATION_OPTIONS
-            -Wno-ignored-attributes
-            -Wno-deprecated-declarations
-            CACHE STRING "AMD OpenCL SDK Compilation Option Requirements")
-        message ("OpenCL support found in the AMD SDK")
-    endif ()
+find_package (OpenCL)
```

Comment:
This doesn't work if you manually installed the SDK (as I had to do on VMWare), as well as requiring a PACKAGE_CMake.txt file to exist in the search path.

Reply:
Restored.